packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -100,7 +100,13 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
     ]);
 
     return (
-      <div role="search" ref={this._rootElement} className={classNames.root} onFocusCapture={this._onFocusCapture}>
+      <div
+        role="search"
+        ref={this._rootElement}
+        className={classNames.root}
+        onBlurCapture={this._onBlur}
```

Comment:
Why the capture version?

Reply:
we need to capture blur whether it happens on the input or the button. I could add onBlur to both, but that seems inconsistent.

Reply:
No I mean I'm curious why you needed to catch it specifically in the capture phase

Reply:
oh, i didn't think about the different phases in this case. Added it to be consistent with focus being at that level, and reduce redundant onBlur I would have had to put onto the button. I can go either way.